measures/2019/measures-schema.yaml

```diff
@@ -30,158 +37,183 @@ definitions:
       measureSpecification:
         description: URL link for Measure Specification PDF to download by Submission Method.
         items: { $ref: #/definitions/measureSpecification }
+      isClinicalGuidelineChanged:
```

Comment:
I thought I only saw this on quality measures, is it true that it applies to all measures?

Reply:
Ah, yes, you're right. Good catch. Fixed in e287e6e.